docs/claws-mail.md

```diff
@@ -0,0 +1,72 @@
+# vdirsyncer with Claws Mail
+
+First of all, you must know that you can **not** edit CardDav contacts only **watch** CardDav contacts with Claws-Mail
+
+## Preparation
+First of al you need to install vdirsyncer, for that look [here](https://vdirsyncer.pimutils.org/en/stable/installation.html).
+Then we need to create some folders:
+
+    mkdir -p ~/.vdirsyncer/status
+    mkdir ~/.contacts
```

Comment:
I think it should be only necessary to create the `.vdirsyncer` folder, all other folders are created automatically (or rather, vdirsyncer asks).